web/website.ini

```diff
@@ -9,7 +9,7 @@ Authors: "Andreas Rumpf and contributors"
 # Everything after ; is the ID
 Community: "community.html;link_forum"
 Aporia_IDE: "https://github.com/nim-lang/Aporia;link_aporia"
-Github_Repo: "http://github.com/Araq/Nim;link_github"
+GitHub_Repo: "http://github.com/Araq/Nim;link_github"
```

Comment:
That looks like it's referenced somewhere, could you check it doesn't break anything?